cmake/configs/posix_sitl_default.cmake

```diff
@@ -165,7 +165,7 @@ set(config_module_list
 	#
 
 	# Tutorial code from
-	# https://px4.io/dev/px4_simple_app
+	# https://dev.px4.io/en/tutorials/tutorial_hello_sky.html
 	examples/px4_simple_app
 
 	# Tutorial code from
```

Comment:
@bkueng 
- The other examples don't appear in the devguide. Do you know where they are used/documented and whether they are useful?
- Almost every other cmake file includes these, but I won't update until clear picture of what needs to happen to the other examples.

Reply:
- px4_daemon_app: this would need to be significantly updated (C++, to use the module base class, ...). But I do think something like that would be helpful. It can serve as a template for new (standard) modules (and at the same time explaining some of the used coding style). If you want me to do that I can take it on my list.
- px4_mavlink_debug: probably don't need an example app, but we could explain the uorb topics for that. I can see that it's already there: https://dev.px4.io/en/debug/debug_values.html
- fixedwing_control: this can be useful, but I think just adding a link from the dev-guide to the source should be enough, since it already contains verbose comments.

Reply:
> px4_daemon_app: this would need to be significantly updated (C++, to use the module base class, ...). But I do think something like that would be helpful. It can serve as a template for new (standard) modules (and at the same time explaining some of the used coding style). If you want me to do that I can take it on my list.

Yes, can you please add that to your list. Lets do it right :-)

Approximate ETA?

> px4_mavlink_debug: probably don't need an example app, but we could explain the uorb topics for that. I can see that it's already there: https://dev.px4.io/en/debug/debug_values.html

As the example already exists no point removing it. Will just update the link to https://dev.px4.io/en/debug/debug_values.html as you indicated.

> fixedwing_control: this can be useful, but I think just adding a link from the dev-guide to the source should be enough, since it already contains verbose comments.

We don't really have anywhere good to create such a link because we don't have much controller documentation. The only current options I see are:
-  up the top of https://dev.px4.io/en/flight_stack/controller_diagrams.html 
- Where you define controllers here: https://dev.px4.io/en/concept/architecture.html#flight-stack

What would be better is if we provided a more detailed overview of how controllers work and link from that. This might reasonably live in Concepts. It might well be a "worked tutorial" of this example. Thoughts?

In short term we could perhaps link to the architecture page from the example?